pages/auth/index.tsx

```diff
@@ -131,43 +133,45 @@ export default function AuthPage() {
-                    {signInOption ? 'Sign in' : 'Create an account'}
+                  <h1 className="md:text-3xl text-2xl text-center text-primaryDark mt-4 poppins-bold">
+                    {signInOption ? 'Login' : 'Create an account'}
                   </h1>
-                  <div className="text-center text-complementary/60 mt-4 mb-12">
-                    {signInOption ? ' New to HackPortal?' : 'Already have an account?'}{' '}
+                  <div className="text-center text-complementary/60 mt-4 mb-12 poppins-semibold">
+                    {signInOption ? ' New to HackUTD?' : 'Already have an account?'}{' '}
                     <span
                       onClick={() =>
                         signInOption ? setSignInOption(false) : setSignInOption(true)
                       }
-                      className="text-primary cursor-pointer"
+                      className="text-primaryDark cursor-pointer underline"
                     >
-                      {signInOption ? 'Create an account' : 'Sign in'}
+                      {signInOption ? 'Register here!' : 'Sign in'}
                     </span>
                   </div>
                   <React.Fragment>
                     <form onSubmit={handleSubmit} className="mt-4">
-                      <input
-                        className="w-full rounded-md border border-complementary/20 p-2 mb-4"
-                        value={currentEmail}
+                      <EmailInput
                         onChange={(e) => setCurrentEmail(e.target.value)}
-                        type="text"
-                        name="email"
-                        autoComplete="email"
-                        placeholder="Email Address*"
-                      ></input>
-                      <input
-                        className="w-full rounded-md border border-complementary/20 p-2 mb-2"
+                        placeholder="Enter your email address"
+                        value={currentEmail}
+                      />
+                      <PasswordInput
                         value={currentPassword}
                         onChange={(e) => setCurrentPassword(e.target.value)}
-                        type={showPassword ? 'text' : 'password'}
-                        name="password"
-                        autoComplete="current-password"
-                        placeholder="Password*"
-                      ></input>
-                      <div className="inline-flex md:flex justify-between md:flex-row flex-col-reverse">
+                        placeholder="Enter your password"
+                      />
+
+                      <div className="inline-flex md:flex justify-between md:flex-row flex-col-reverse poppins-semibold text-sm">
+                        <div className="text-primaryDark">
+                          {/* TODO: should this feature be implemented or does it already exist? */}
```

Comment:
I believe session will persist as long as you don't sign out. This is a great-to-have feature in my opinion, so it is totally optional whether you want to implement this or not.